### DISCUSSION TITLE
Post processing g-code - from Simplify3D

### DISCUSSION THREAD

**Discussion by Denniswie** [supermerill/SuperSlicer#1733] [Q&A] [answered]
I am currently trying out Superslicer, and thing I will permanently change over.
I have a post script to save the g-code to my Octoprint (192.168....)
-----------------------------
curl -k -H "X-Api-Key: D8E83680A4684E43A3043CEAE351B4D8" -F "select=false" -F "print=false" -F "file=@[output_filepath]" "http://192.168.1.55:80/api/files/local" {STRIP "; postProcessing"};
----------------------------------
but this does not work in SuperSlicer.

Does it support this type of functionality ?
Could you suggest a good wiki to help me understand the process involved above, and modify to suit SuperSlicer ?

**Comment by supermerill** [comment, marked as answer]
You can register your octoprint server so you can upload your gcode directly from the slicer.
on your printer profile, click on the cog, at the top.
select octoprint, enter the IP, the API key. test it.
If you want to use multiple printer profiles, use the (+) to add them.
push ok. Now when your object is slice, you have a "send" button. It will open a window where you set the name of the file for octorpint, and if it has to start printing immediately.

If you want to use a post-process script, it's in print->output. The command will be called with the gcode file path as an argument.
